storage/resource_databricks_files.go

```diff
@@ -0,0 +1,127 @@
+package storage
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"crypto/md5"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/databricks/databricks-sdk-go/service/files"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/databricks/terraform-provider-databricks/workspace"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func getContentReader(data *schema.ResourceData) (io.ReadCloser, error) {
```

Comment:
Maybe it makes sense to unify it with https://github.com/databricks/terraform-provider-databricks/blob/master/workspace/file_resource.go#L36? That code is also calculating the MD5 of the source file...
Although we can do it at the later point of time

Reply:
Yeah, it would need some refactoring, I will try to do it when migrating other resources which will also require some refactoring so will do them simultaneously.